app/code/Magento/InventoryCatalog/Plugins/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php

```diff
@@ -4,21 +4,20 @@
  * See COPYING.txt for license details.
  */
 
-namespace Magento\InventoryCatalog\Plugin\Model;
+namespace Magento\InventoryCatalog\Plugins\Inventory\Reservation;
```

Comment:
Generally in Magento plugins directory should be `Plugin` (single form but not plurar)

Reply:
This refactoring has been performed. Thank you!